### ISSUE TITLE
Linux's Version Keeps Giving Me An Error Message

### ISSUE THREAD

**Issue by DarkestAngelVT** [KevinBytesTheDust/CVAmp#284]
Here's the error message

[pid=123749][err] /opt/google/chrome/chrome: symbol lookup error: /lib/x86_64-linux-gnu/libharfbuzz.so.0: undefined symbol: FT_Get_Transform died at page None

I'm not sure why it's chrome causing the problem because I have other apps that use chrome the same way this does and they connect just fine so something along the way from an update got broken with the chrome connection.

Any help with this problem would be appreciated.

**Comment by jlplenio** [comment]
Could you try this? `sudo apt-get install --reinstall libfreetype6 libharfbuzz0b`

**Comment by DarkestAngelVT** [comment]
> Could you try this? `sudo apt-get install --reinstall libfreetype6 libharfbuzz0b`

That fixed one problem but now it showed this error code:

[pid=16289][err] /opt/google/chrome/chrome: symbol lookup error: /lib/x86_64-linux-gnu/libharfbuzz.so.0: undefined symbol: FT_Get_Transform
2024-07-19 13:39:24,530;INFO;20_8;Thread-2 (spawn_instance_thread);manager;instance_status_report_callback;SHUTDOWN instance 1

**Comment by jlplenio** [comment]
What distribution do you use?

**Comment by DarkestAngelVT** [comment]
> What distribution do you use?

Ubuntu

**Comment by jlplenio** [comment]
Which version?
Sorry, I am shooting in the dark here.
1. Can you try to update our chrome to the latest version?
2. Can you try to downgrade libharfbuzz0b with sudo apt install libharfbuzz0b=2.7.4-1+b1

**Comment by DarkestAngelVT** [comment]
I'm on Ubuntu 24.04 LTS

My chrome is the latest version I always check for updates for my software and apps everyday

I put in the command ' sudo apt install libharfbuzz0b=2.7.4-1+b1 ' into the terminal and I'm getting this error message:

$ sudo apt install libharfbuzz0b=2.7.4-1+b1
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libharfbuzz0b is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libharfbuzz0b:i386

**Comment by jlplenio** [comment]
Did you run `sudo apt update` beforehand?
Can you try to install the newest version with `sudo apt install libharfbuzz0b`?
Can you check which version is currently installed with `apt-cache policy libharfbuzz0b`?

**Comment by DarkestAngelVT** [comment]
Did you run sudo apt update beforehand?
Yes I did, I always run sudo apt update after updating my system / apps

![Terminal Command](https://github.com/user-attachments/assets/9da8fcf6-3c4e-4e83-a115-43400bc13dc4)

To my understanding I have the package already so nothing has changed

**Comment by jlplenio** [comment]
I am simply guessing at this point. Can you easily switch distros or versions, like going to 22.04?

**Comment by DarkestAngelVT** [comment]
> I am simply guessing at this point. Can you easily switch distros or versions, like going to 22.04?

I can do that, I'm just testing this on my side machine so we can try a few things since no data will be lost if anything goes wrong, I'll install it and keep you posted

**Comment by DarkestAngelVT** [comment]
> I am simply guessing at this point. Can you easily switch distros or versions, like going to 22.04?

Okay so going under 22.04.4 worked now the bots connect just fine, I'm not sure why 24.04 doesn't work 
A Side note is I ran ' sudo apt install libharfbuzz0b ' after the install I don't think it did anything but just something I wanted to add in the steps I took.

I want to thank you for helping me on this.

I hope one day they can work for 24.04 because most people when they google ' Linux ubuntu download  ' will get brought to 24.04 download page and some people don't know they have an alt downloads page, either way still cool it works.
